routes/products.js

```diff
@@ -0,0 +1,60 @@
+let express = require('express');
+let router = express.Router();
+const Products = require('../db/products');
+const exphbs = require('express-handlebars');
+const app = express();
+
+
+router.get('/', (req,res) => {
+  res.render('./productViews/indexPro', {products : Products.getAllProducts()});
+});
+
+router.get('/new', (req,res) => {
+  res.render('./productViews/newPro', (err, hbs) => {
```

Comment:
Not sure if this is recommended. You could just leave the second parameter blank I believe.